hack/verify-bazel.sh

```diff
@@ -21,11 +21,9 @@ KOPS_ROOT=$(git rev-parse --show-toplevel)
 TMP_GOPATH=$(mktemp -d)
 cd "${KOPS_ROOT}"
 
-"${KOPS_ROOT}/hack/go_install_from_commit.sh" \
-  github.com/bazelbuild/bazel-gazelle/cmd/gazelle \
-  a85b63b06c2e0c75931e57c4a1a18d4e566bb6f4 \
-  "${TMP_GOPATH}"
+ln -s ${KOPS_ROOT}/vendor/ ${TMP_GOPATH}/src
```

Comment:
Make sure we do the update script as well please

Reply:
Good call - done